index.html

```diff
@@ -9,9 +9,9 @@
     <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
 </head>
 <body>
-<div class="page1">
+<section class="page1">
     <header>
-    <a href="index.html">Philip Modinos</a>
+    <h1>Philip Modinos</h1>
```

Comment:
- [ ] Good job on trying to make some changes to your project. However, you seem to have broken something here. I think you forgot to wrap the `h1` element with an anchor (`a`) tag. This will help in making your name a link.